drivers/packet/create.go

```diff
@@ -5,15 +5,89 @@
 package packet
 
 import (
+	"bytes"
 	"context"
+	"fmt"
+	"strings"
+	"time"
 
 	"github.com/drone/autoscaler"
+	"github.com/packethost/packngo"
```

Comment:
This package have to be added to Gopkg.toml

Reply:
it was already there